mission/config/ui/taskroster/playerInfo/ui.hpp

```diff
@@ -0,0 +1,546 @@
+		};
+
+		class rhs_img_dutyofficer_wheel_caption: rhs_img_dutyofficer_map_caption
+		{
+			x = UIW(13.5);
+			text = "Wheel Menu";
+		};
+	};
+};
+
+class vn_tr_disp_showPlayerInfo
+{
+	name = "vn_tr_disp_showPlayerInfo";
+	//If already opened -> Recalling it -> Reloading the Dialog (e.g. like updating the view, without "closing" it)
+	onLoad = "[""onLoad"",_this,""vn_tr_disp_showPlayerInfo"",''] call (uinamespace getvariable 'BIS_fnc_initDisplay'); call vn_mf_fnc_tr_playerInfo_show;";
+	onUnload = "[""onUnload"",_this,""vn_tr_disp_showPlayerInfo"",''] call (uinamespace getvariable 'BIS_fnc_initDisplay'); [] spawn vn_mf_fnc_tr_overview_init;";
+	idd = THISISWRONGONPURPOSE;
+	movingEnable = 1;
+	enableSimulation = 1;
+
+	class ControlsBackground
+	{
+		class folderBackground: vn_mf_RscPicture
+		{
+			idc = -1;
+			x = VN_TR_FOLDER_X;
+			y = VN_TR_FOLDER_Y;
+			w = VN_TR_FOLDER_W;
+			h = VN_TR_FOLDER_H;
+			colorText[] = {1,1,1,1};
+			colorBackground[] = {1,1,1,1};
+			text = "\vn\ui_f_vietnam\ui\taskroster\img\tr_folder_background_sheetL.paa";
+		};
+
+		//MUST be in the background, otherwise it could "pop up", when clicking on it, covering up the cordles
+		class rhs: vn_tr_disp_showPlayerInfo_rhs
+		{
+			idc = VN_TR_PLAYERINFO_RHS_IDC;
+		};
+	};
+
+	class Controls
+	{
+		class title: vn_mf_RscText
+		{
+			idc = -1;
+			x = UIW(0);
+			y = UIH(2);
+			w = UIW(15);
+			h = UIH(2);
+
+			style = "0x10 + 0x0200";
+			colorText[] = {0.1,0.1,0.1,0.9};
+			colorBackground[] = {0,0,0,0.0};
+			shadow = 0;
+			text = "Your Info/Stats";
+			font = USEDFONT_B;
+			sizeEx = TXT_L;
+		};
+
+		class playername: vn_mf_RscStructuredText
+		{
+			idc = VN_TR_PLAYERINFO_NAME_IDC;
+			x = UIW(0);
+			y = UIH(4);
+			w = UIW(15);
+			h = UIH(0.8);
+
+			style = "0x10 + 0x0200";
+			colorText[] = {0.1,0.1,0.1,0.9};
+			colorBackground[] = {0,0,0,0.0};
+			shadow = 0;
+			text = "playername";
+			font = USEDFONT;
+			sizeEx = TXT_S;
+			tooltip = "Player Name";
+			class Attributes
+			{
+				align = "left";
+				color = "#000000";
+				colorLink = "#D09B43";
+				font = USEDFONT;
+				size = 0.85;
+				shadow = 0;
+			};
+		};
+		class serialnumber: playername
+		{
+			idc = VN_TR_PLAYERINFO_SNUM_IDC;
+			y = UIH(5);
+			text = "serialnumber";
+			tooltip = "Serial Number";
+		};
+		class playeruid: playername
+		{
+			idc = VN_TR_PLAYERINFO_UID_IDC;
+			y = UIH(6);
+			text = "playeruid";
+			tooltip = "Player UID";
+		};
+		class playerrank: playername
+		{
+			idc = VN_TR_PLAYERINFO_RANK_IDC;
+			y = UIH(7);
+			text = "playerrank";
+			tooltip = "Player Rank";
+		};
+		class killtracking: playername
+		{
+			idc = VN_TR_PLAYERINFO_KILLS_IDC;
+			y = UIH(8);
+			text = "sessionkills";
+			tooltip = "Session Kills";
+		};
+		class deaths: playername
+		{
+			idc = VN_TR_PLAYERINFO_DEATHS_IDC;
+			y = UIH(9);
+			text = "-1";
+			tooltip = "Session Deaths";
+		};
```

Comment:
Delete this, not doing session kills so probably don't want session deaths either